config/webpack/common.js

```diff
@@ -63,7 +64,10 @@ export default {
     new CopyWebpackPlugin([{
       from: './favicon'
     }]),
-    new webpack.optimize.OccurenceOrderPlugin()
+    new webpack.optimize.OccurenceOrderPlugin(),
+    new Dotenv({
+      systemvars: true  // Variables from npm scripts and from CI pipeline
```

Comment:
When building for prod this plugin outputs an error that `./.env is not found`, maybe also set `silent: true`?

Reply:
a7ba4241bb0abd3089ed50ec442ad2740aacf285